pkg/services/queryhistory/writers.go

```diff
@@ -23,12 +23,12 @@ func writeStarredSQL(query SearchInQueryHistoryQuery, sqlStore *sqlstore.SQLStor
 }
 
 func writeFiltersSQL(query SearchInQueryHistoryQuery, user *models.SignedInUser, sqlStore *sqlstore.SQLStore, builder *sqlstore.SQLBuilder) {
-	params := []interface{}{user.OrgId, user.UserId, "%" + query.SearchString + "%"}
+	params := []interface{}{user.OrgId, user.UserId, "%" + query.SearchString + "%", "%" + query.SearchString + "%"}
 	for _, uid := range query.DatasourceUIDs {
 		params = append(params, uid)
 	}
 	var sql bytes.Buffer
-	sql.WriteString(" WHERE query_history.org_id = ? AND query_history.created_by = ? AND query_history.queries " + sqlStore.Dialect.LikeStr() + " ? AND query_history.datasource_uid IN (? " + strings.Repeat(",?", len(query.DatasourceUIDs)-1) + ") ")
+	sql.WriteString(" WHERE query_history.org_id = ? AND query_history.created_by = ? AND (query_history.queries " + sqlStore.Dialect.LikeStr() + " ? OR query_history.comment " + sqlStore.Dialect.LikeStr() + " ?) AND query_history.datasource_uid IN (? " + strings.Repeat(",?", len(query.DatasourceUIDs)-1) + ") ")
```

Comment:
I was also thinking about performance. Over time, we don't want any such search to depend on SQL but rather have a search service for this (so we can support other stores and whatnot), but until then we might have to.

I don't think an index would help here, since we're looking for the search string in an arbitrary position within the column 🤔

Reply:
> I don't think an index would help here, since we're looking for the search string in an arbitrary position within the column

right, I thought the pattern was `<prefix>%`